### PR TITLE
Bump PLONE_BACKEND_PATCH_VERSION for docker image to 6.0.10.1

### PR DESCRIPTION
**Pull request by stevepiercy**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -341,7 +341,7 @@ def source_replace(app, docname, source):
 # Dict of replacements.
 source_replacements = {
     "{PLONE_BACKEND_MINOR_VERSION}": "6.0",
-    "{PLONE_BACKEND_PATCH_VERSION}": "6.0.9",
+    "{PLONE_BACKEND_PATCH_VERSION}": "6.0.10.1",
     "{NVM_VERSION}": "0.39.5",
     "{SUPPORTED_PYTHON_VERSIONS}": "3.8, 3.9, 3.10, or 3.11",
```
Comment (mauritsvanrees) [inline docs/conf.py]: We can add 3.12 now:

```suggestion
    "{SUPPORTED_PYTHON_VERSIONS}": "3.8, 3.9, 3.10, 3.11, or 3.12",
```

**Review by mauritsvanrees — CHANGES_REQUESTED** [review]
LGTM, once Docker is ready.

I added one additional suggestion.

**Comment by stevepiercy** [comment]
We can ignore the failed linkcheck. I have an open PR for one of the links, and Linode appears to disallow HEAD requests again.

**Comment by ericof** [comment]
@stevepiercy Images being built on https://github.com/plone/plone-backend/actions/runs/8088441702

**Comment by stevepiercy** [comment]
image

**Pull request merged**